src/workspaces/list/NoContentMessage.tsx

```diff
@@ -17,15 +14,11 @@ export const NoContentMessage = (props: NoContentMessageProps): ReactNode => {
   const { loadingWorkspaces, workspaces, filters } = props;
   const { setUserActions } = useContext(WorkspaceUserActionsContext);
 
-  return cond(
-    [loadingWorkspaces, () => h(Fragment, ['Loading...'])],
-    [
-      _.isEmpty(workspaces.myWorkspaces) && filters.tab === 'myWorkspaces',
-      () =>
-        NoWorkspacesMessage({
-          onClick: () => setUserActions({ creatingNewWorkspace: true }),
-        }),
-    ],
-    () => div({ style: { fontStyle: 'italic' } }, ['No matching workspaces'])
-  );
+  if (loadingWorkspaces) {
+    return <> Loading... </>;
+  }
+  if (workspaces.myWorkspaces.length === 0 && filters.tab === 'myWorkspaces') {
```

Comment:
One thing to be careful about is whether you can guarantee that `workspaces.myWorkspaces` is not undefined/null. `_.isEmpty` can handle null/undefined (returning true).

Reply:
The conditonal operator (`?.`) and defaults (`??`) might help here (`workspaces?.myWorkspaces?.length ?? 0`).